rules/linux/command_and_control_linux_kworker_netcon.toml

```diff
@@ -0,0 +1,76 @@
+[metadata]
+creation_date = "2023/10/18"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "Multiple field support in the New Terms rule type was added in Elastic 8.6"
+min_stack_version = "8.6.0"
+updated_date = "2023/10/18"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule monitors for network connections from a kworker process. kworker, or kernel worker, processes are part of the
+kernel's workqueue mechanism. They are responsible for executing work that has been scheduled to be done in kernel 
+space, which might include tasks like handling interrupts, background activities, and other kernel-related tasks. 
+Attackers may attempt to evade detection by masquerading as a kernel worker process.
+"""
+from = "now-59m"
+index = ["logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License v2"
+name = "Network Activity Detected via kworker"
+risk_score = 21
+rule_id = "25d917c4-aa3c-4111-974c-286c0312ff95"
+severity = "low"
+tags = [
+        "Domain: Endpoint",
+        "OS: Linux",
+        "Use Case: Threat Detection",
+        "Tactic: Command and Control",
+        "Data Source: Elastic Defend"
+        ]
+timestamp_override = "event.ingested"
+type = "new_terms"
+
+query = '''
+host.os.type:linux and event.category:network and event.action:(connection_attempted or connection_accepted) and 
+process.name:kworker*
+'''
+
+[[rule.threat]]
+framework = "MITRE ATT&CK"
+
```

Comment:
Would be nice if we could find techniques to include here and in the Exfiltration tactic